src/components/ContentNode.vue

```diff
@@ -297,18 +307,29 @@ function renderNode(createElement, references) {
         renderChildren(node.inlineContent)
       ));
     }
-    case BlockType.table:
+    case BlockType.table: {
+      let tableContent = [renderTableChildren(
+        node.rows, node.header, node.extendedData, node.alignments,
+      )];
+
       if (node.metadata && node.metadata.anchor) {
-        return renderFigure(node);
+        tableContent = placeCaption(renderTableChildren(
+          node.rows, node.header, node.extendedData, node.alignments,
+        ), {
+          title: node.metadata.title,
+          abstract: node.metadata.abstract,
```

Comment:
I would probably pass the `tag` here explicitly, just so its obvious.

Reply:
okay, I did it here https://github.com/apple/swift-docc-render/pull/692/commits/2d2864bdedd0c0e9a6eec699493e1d91bd1c919d